doc/manual/src/installation/installing-binary.md

```diff
@@ -1,89 +1,112 @@
+The corresponding SHA-256 hash can be found in the directory for the given version.
+
+These installation scripts can be used the same as usual:
 
 ```console
-$ mkdir /nix
-$ chown alice /nix
+$ VERSION=2.19.2 curl -L https://releases.nixos.org/nix/nix-$VERSION/install | sh
 ```
 
-The install script will modify the first writable file from amongst
-`.bash_profile`, `.bash_login` and `.profile` to source
-`~/.nix-profile/etc/profile.d/nix.sh`. You can set the
-`NIX_INSTALLER_NO_MODIFY_PROFILE` environment variable before executing
-the install script to disable this behaviour.
-
 # Multi User Installation
 
-The multi-user Nix installation creates system users, and a system
-service for the Nix daemon.
+The multi-user Nix installation creates system users, and a system service for the Nix daemon.
+
+Supported systems:
 
-**Supported Systems**
 - Linux running systemd, with SELinux disabled
 - macOS
 
-You can instruct the installer to perform a multi-user installation on
-your system:
+You can explicitly instruct the installer to perform a multi-user installation on your system:
 
 ```console
 $ curl -L https://nixos.org/nix/install | sh -s -- --daemon
 ```
 
-The multi-user installation of Nix will create build users between the
-user IDs 30001 and 30032, and a group with the group ID 30000. You
-can run this under your usual user account or root. The script
-will invoke `sudo` as needed.
+You can run this under your usual user account or `root`.
+The script will invoke `sudo` as needed.
+
+# Single User Installation
+
+To explicitly select a single-user installation on your system:
+
+```console
+$ curl -L https://nixos.org/nix/install | sh -s -- --no-daemon
+```
+
+This will perform a single-user installation of Nix, meaning that `/nix` is owned by the invoking user.
+You can run this under your usual user account or `root`.
+The script will invoke `sudo` to create `/nix` if it doesn’t already exist.
+If you don’t have `sudo`, manually create `/nix` as `root`:
+
+```console
+$ su root
+# mkdir /nix
+# chown alice /nix
+```
+
+# Installing from a binary tarball
+
+You can also download a binary tarball that contains Nix and all its dependencies:
+- Choose a [version](https://releases.nixos.org/?prefix=nix/) and [system type](../contributing/hacking.md#platforms)
+- Download and unpack the tarball
+- Run the installer
 
-> **Note**
+> **Example**
 >
-> If you need Nix to use a different group ID or user ID set, you will
-> have to download the tarball manually and [edit the install
-> script](#installing-from-a-binary-tarball).
+> ```console
+> $ pushd $(mktemp -d)
+> $ export VERSION=2.19.2
+> $ export SYSTEM=x86_64-linux
+> $ curl -LO https://releases.nixos.org/nix/nix-$VERSION/nix-$VERSION-$SYSTEM.tar.xz
+> $ tar xfj nix-$VERSION-$SYSTEM.tar.xz
+> $ cd nix-$VERSION-$SYSTEM-darwin
+> $ ./install
+> $ popd
+> ```
+
+The installer can be customised using environment varibales set in the file named `install-multi-user`.
```

Comment:
Your suggestion merely switches to "you can" active voice, it doesn't change the framing. Maybe the wording is potentially misleading, because those env vars *are* set in the file, but it may be read as "which *you* set (wherever)". 

```suggestion
The installer can be customised using environment variables declared in the file named `install-multi-user`.
```